napari/_qt/qt_main_window.py

```diff
@@ -898,6 +924,18 @@ def _add_help_menu(self):
         )
         self.help_menu.addAction(about_key_bindings)
 
+    def _toggle_activity_dock(self, _):
```

Comment:
`_`?

Reply:
The button click event which is not used. I've changed this to `event` now to be clearer.